Jenkinsfile

```diff
@@ -126,8 +126,8 @@ DFLAGS=-I%@P%/../imports -L-L%@P%/../libs -L--export-dynamic -L--export-dynamic
         '''
         archiveArtifacts artifacts: 'distribution.tar', onlyIfSuccessful: true
     }
-}
 
-stage("Test downstream projects") {
-    build job: 'dlangci-downstream'
 }
+
+// executed from workspace root
+load('dlangci/ProjectsJenkinsfile')
```

Comment:
I've changed it b/c it was a mess to have a loose git folder as root and lots of other projects inside of that.
I'd rather if we put every clone under `ghrepos/` with their github path, e.g. `ghrepos/Dicebot/dlangci`.
That way caching clones will be less confusing if multiple jobs/stages interpose their repo layout.